### ISSUE TITLE
Documentation of @GlobalScope cannot be scrolled down completely

### ISSUE THREAD

**Issue by Chaosus** [godotengine/godot#52678]
### Godot version

0232a031772d929c00d7c8a7947b1b9894ce26fb

### System information

Windows 10

### Issue description

Docs seem to cut down and cannot be scrolled down further there: 

![image](https://user-images.githubusercontent.com/3036176/133313866-89f5a892-f187-4851-8cd1-e97330436ac9.png)

I pretty sure it was reported but don't remember where...

### Steps to reproduce

Open @GlobalScope doc

### Minimal reproduction project

Too trivial to reproduce

**Comment by akien-mga** [comment]
Confirmed on Linux, here I see a tiny bit more of it:
![Screenshot_20210914_211303](https://user-images.githubusercontent.com/4701338/133319612-e382cb87-ae44-426b-ba27-f4c8ebe4e272.png)

But it's longer than that normally, and should include codeblocks. It's the last method in `@GlobalScope`.

Additionally, there's a position bug when clicking a method name in the summary, it doesn't reach the right position. Here, I clicked `seed`:
![Screenshot_20210914_211414](https://user-images.githubusercontent.com/4701338/133319865-17833987-016d-4e14-869f-20574af38a12.png)
And it's actually a double pages down.

I use 100% display scale on 1080p.
